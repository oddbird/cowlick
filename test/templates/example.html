<div class="foo"><a href="http://oddbird.net">Hello, {{org}}!</a> there</div>
