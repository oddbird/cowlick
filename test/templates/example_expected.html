<div class="foo"><a href="http://oddbird.net">Hello, OddBird!</a> there</div>
